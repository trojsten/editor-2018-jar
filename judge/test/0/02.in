VECTOR_INT:
vector_intov
0
INT:
intt
0
VECTOR_STR:
vector_stringov
0
STR:
stringg

VECTOR_FLOAT:
vector_floatov
0
FLOAT:
floatik
0
