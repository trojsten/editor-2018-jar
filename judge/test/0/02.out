{"floatik": 0.333333}
